samples/dotnetapp/README.md

```diff
@@ -105,13 +105,13 @@ Note: The `/p:Configuration=Release` argument builds the application in release
 More Samples
 
 * [.NET Framework Docker Samples](../README.md)
-* [.NET Core Docker Samples](https://github.com/dotnet/dotnet-docker/blob/master/samples/README.md)
+* [.NET Docker Samples](https://github.com/dotnet/dotnet-docker/blob/master/samples/README.md)
 
 Docs and More Information:
 
 * [.NET Docs](https://docs.microsoft.com/dotnet/)
 * [ASP.NET Docs](https://docs.microsoft.com/aspnet/)
-* [dotnet/core](https://github.com/dotnet/core) for starting with .NET Core on GitHub.
```

Comment:
I'm not sure the link should be changed here.  `dotnet/core` is the "starting" repository.  It's readme even says 

> The dotnet/core repository is a good starting point for .NET Core.

This link appears in several places.

Reply:
Yep, that was not intentional.  A little too aggressive with find-and-replace.